### ISSUE TITLE
Inconsistent DB scheme 

### ISSUE THREAD

**Issue by dasistwas** [Wunderbyte-GmbH/moodle-mod_booking#188]
Runnding check database results in these errors:

```
-------------------------------------------------------------------------------
booking_options
 * column 'parentid' is missing
 * Missing index 'parentid' (not unique (parentid)).
CREATE INDEX mdl_bookopti_par_ix ON mdl_booking_options (parentid);
-------------------------------------------------------------------------------
booking_icalsequence
 * column 'userid' should be NOT NULL (I)
 * column 'userid' has default '0', expected 'NULL' (I)
 * column 'optionid' should be NOT NULL (I)
 * column 'optionid' has default '0', expected 'NULL' (I)
 * column 'sequencevalue' should be NOT NULL (I)
 * column 'sequencevalue' has default '0', expected 'NULL' (I)
-------------------------------------------------------------------------------
booking_userevents
 * column 'userid' has default '0', expected 'NULL' (I)
 * column 'optionid' has default '0', expected 'NULL' (I)
 * column 'optiondateid' has default '0', expected 'NULL' (I)
 * column 'eventid' has default '0', expected 'NULL' (I)

```

Found on bipedu....

**Comment by dasistwas** [comment]
```

$plugin->version = 2021111608;
$plugin->requires = 2019111800; // Requires this Moodle version. Current: Moodle 3.8.
$plugin->release = 'v7.0.20.06';
$plugin->maturity = MATURITY_STABLE;
$plugin->cron = 60;
$plugin->component = 'mod_booking';

```

**Comment by bernhard-wunderbyte** [comment]
Fixed in Version 7.0.20.07 (2021111609)
